.github/workflows/buildAndDeploy.yml

```
@@ -0,0 +1,109 @@
+
+      - name: Build Docker image
+        run: |
+          docker build -t ootd_image:latest docker-context
+          docker tag ootd_image:latest ${{ secrets.AWS_URL }}:latest
+          docker tag ootd_image:latest ${{ secrets.AWS_URL }}:sha-${{ needs.build.outputs.sha_short }}
+
+      - name: Push Docker image to ECR
+        run: |
+          docker push ${{ secrets.AWS_URL }}:latest
+          docker push ${{ secrets.AWS_URL }}:sha-${{ needs.build.outputs.sha_short }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)